tests/openadapt/test_layout.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+from openadapt.tools.layout import document_query
+from PIL import Image
+from transformers import pipeline
+import numpy as np
+import pytest
+
+IMAGE_FILE_NAMES = [
+    "tests/openadapt/assets/test_ladingbill.png",
+    "tests/openadapt/assets/test_invoice.png",
+    "tests/openadapt/assets/test_calendar.png",
+    "tests/openadapt/assets/test_calc.png",
+]
+
+
+def test_lading_bill_screenshot():
+    questions = [
+        "What is the bill of lading number?",
+        "What is the carrier name?",
+        "What is the ship date?",
+        "What is the sender's phone number?",
+    ]
+
+    output_answers = []
+    for q in questions:
+        output = document_query(image=None, image_path=IMAGE_FILE_NAMES[0], question=q)
+        output_answers.append(output[0]['answer'])
+
+    expected_output = [
+        "21099992723",
+        "EXAMPLE CARRIER",
+        "September 13, 2021",
+        "456-7890",
+    ]
+    assert output_answers == expected_output
+
+
+def test_invoice_screenshot():
+    questions = [
+        "What is the sender's phone number?",
+        "What is the sender's address?",
+        "What is surcharge?",
+        "What is the total cost?",
+        "What is the daily interest rate on overdue balances?",
+    ]
+
+    expected_answers = ["321-956-7331,", "MELBOURNE,", "45.00", "$ 3,165.00", "0.5%"]
+
+    output_answers = []
+    for q in questions:
+        output = document_query(image=None, image_path=IMAGE_FILE_NAMES[1], question=q)
+        output_answers.append(output[0]['answer'])
+    assert output_answers == expected_answers
+
+
+def test_calendar_screenshot():
+    output = document_query(
+        image=None, image_path=IMAGE_FILE_NAMES[2], question="What month is it?"
+    )[0]['answer']
+    expected_output = "june"
+    assert output.lower() == expected_output
+
+
+def test_calc_screenshot():
+    output = document_query(
+        image=None,
+        image_path=IMAGE_FILE_NAMES[3],
+        question="What is the current number on the screen?",
+    )
+    expected_output = []
```

Comment:
Can you please confirm that we expect the model to fail to correctly answer the question here?

If that is the case, can you please suggest why this might be the case?

Ideally we would like to figure out how to get it to answer the question (e.g. by modifying the prompt). If we can't do that, we should still modify `expected_output = 10` but have pytest expect an exception here, and add a `TODO` to figure out how to correctly interpret the screenshot.

Reply:
LayoutLM is bad at answering questions on GUI images, which is why we expect it to fail in this case. To be more specific, the pipeline just outputs an empty list as it can't parse the "10" displayed on the screen, or any number for that matter judging by its output. 

As for correctly interpreting the screenshot, that's going to be a little tricky b/c we may get an empty list as in the case of the Calculator screenshot, or some other incorrect answer making it hard to generalize.

Reply:
@FFFiend can you please implement this:

> Ideally we would like to figure out how to get it to answer the question (e.g. by modifying the prompt). If we can't do that, we should still modify expected_output = 10 but have pytest expect an exception here, and add a TODO to figure out how to correctly interpret the screenshot.

Reply:
I'm a bit confused about the exception part. So far I've had the tool return just the list, but not an exception..so Pytest would have to expect something else

Reply:
raise Warning for the test that's supposed to fail